### PR TITLE
Begin Esperanto translations

### PR DESCRIPTION
**Pull request by LorenDB**
I'm not completely sure that I have done everything correctly, but hopefully users will be able to understand. 😁

Stay tuned for more translations!

### CODE REVIEW

File: resources/langs/nheko_eo.ts [hunk 1]
```diff
@@ -0,0 +1,2004 @@
...
+        <source>e.g @joe:matrix.org</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+2"/>
+        <source>Your login name. A mxid should start with @ followed by the user id. After the user id you need to include your server name after a :.
+You can also put your homeserver address there, if your server doesn&apos;t support .well-known lookup.
+Example: @user:server.my
+If Nheko fails to discover your homeserver, it will show you a field to enter the server manually.</source>
+        <translation type="unfinished">Via ensaluta nomo.</translation>
```
Comment (deepbluev7) [inline resources/langs/nheko_eo.ts:361]: This looks quite a bit too short?

File: resources/langs/nheko_eo.ts [hunk 2]
```diff
@@ -0,0 +1,2004 @@
...
+        <source>no version stored</source>
+        <translation type="unfinished"></translation>
+    </message>
+</context>
+<context>
+    <name>RoomInfoListItem</name>
+    <message>
+        <location filename="../../src/RoomInfoListItem.cpp" line="+102"/>
+        <source>Leave room</source>
+        <translatorcomment>Maybe shouldn&apos;t be imperative case?</translatorcomment>
```
Comment (deepbluev7) [inline resources/langs/nheko_eo.ts:614]: I have no idea, what is usually used in Esperanto :D

File: resources/langs/nheko_eo.ts [hunk 3]
```diff
@@ -0,0 +1,2004 @@
...
+    </message>
+    <message>
+        <location line="+1"/>
+        <source>Received</source>
+        <translation>Estas ricevita</translation>
+    </message>
+    <message>
+        <location line="+1"/>
+        <source>Read</source>
+        <translatorcomment>Could this simply be &quot;lega&quot;?</translatorcomment>
```
Comment (deepbluev7) [inline resources/langs/nheko_eo.ts:724]: Without knowing anything about Esperanto, I'd say yes.

File: resources/langs/nheko_eo.ts [hunk 4]
```diff
@@ -0,0 +1,2004 @@
...
+    <message>
+        <location line="+945"/>
+        <source>Rejected the knock from %1.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+2"/>
+        <source>%1 left after having already left!</source>
+        <comment>This is a leave event after the user already left and shouldn&apos;t happen apart from state resets</comment>
+        <translation type="unfinished">%1 foriris post jam forirado</translation>
```
Comment (deepbluev7) [inline resources/langs/nheko_eo.ts:947]: Missing the `!`?

**Review by deepbluev7 — COMMENTED** [review]
It's a bit hard to review, if you don't know the language, but I think I found a few small issues?

File: resources/langs/nheko_eo.ts (hunk 1 above)
Comment (LorenDB) [reply resources/langs/nheko_eo.ts:361]: Notice: type="unfinished"

File: resources/langs/nheko_eo.ts (hunk 4 above)
Comment (LorenDB) [reply resources/langs/nheko_eo.ts:947]: Notice: type="unfinished"

File: resources/langs/nheko_eo.ts (hunk 1 above)
Comment (deepbluev7) [reply resources/langs/nheko_eo.ts:361]: Ah, right, didn't see that.

File: resources/langs/nheko_eo.ts (hunk 4 above)
Comment (deepbluev7) [reply resources/langs/nheko_eo.ts:947]: Still, in this case it is only missing the !, isn't it?

Comment (LorenDB) [reply resources/langs/nheko_eo.ts:947]: No, it only goes to the "... user already left".

Comment (LorenDB) [reply resources/langs/nheko_eo.ts:947]: Plus I'm not sure if what I have is even right :/

Comment (deepbluev7) [reply resources/langs/nheko_eo.ts:947]: Ah, I see :D

**Comment by deepbluev7** [comment]
Do you want to work on this more or should I merge it as is?

**Comment by LorenDB** [comment]
I don't care.
________________________________

**Comment by deepbluev7** [comment]
Then ping me, when I should merge it :3

**Comment by LorenDB** [comment]
@deepbluev7 Why don't you just go ahead and merge it? That way we've got some translations in there :)

**Pull request merged**